### PR TITLE
JS Sample update on Guaranteed Publisher/Subscriber and Queue Producer/Consumer

### PR DESCRIPTION
**Pull request by gvensan**
The goal is to bring JS samples on par with JCSMP samples 
- Guaranteed publisher/Subscriber 
- Queue producer/consumer

Changes:
  - Modified QueueProducer & QueueConsumer to work with Topic subscriptions
  - Moved QueueProducer & Consumer to features
  - Added Guaranteed Publisher & Subscriber aligned with convention-based topic solace/samples/js/pers/pub/*
Updated the HTML & CSS with additional instruction (pre-requisite) and a clear button to unclutter the textarea

### CODE REVIEW

File: src/basic-samples/GuaranteedPublisher/GuaranteedPublisher.js [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+        var time = [('0' + now.getHours()).slice(-2), ('0' + now.getMinutes()).slice(-2),
+            ('0' + now.getSeconds()).slice(-2)];
+        var timestamp = '[' + time.join(':') + '] ';
+        console.log(timestamp + line);
+        var logTextArea = document.getElementById('log');
+        logTextArea.value += timestamp + line + '\n';
+        logTextArea.scrollTop = logTextArea.scrollHeight;
+    };
+
+    publisher.log('\n*** publisher to topic "' + publisher.topicName + '/*" is ready to connect ***');
```
Comment (TamimiGitHub) [inline src/basic-samples/GuaranteedPublisher/GuaranteedPublisher.js:47]: no need to add `/*` in the log

File: src/features/QueueConsumer/QueueConsumer.html [hunk 2]
```diff
@@ -142,17 +143,20 @@ <h3>Queue Consumer</h3>
           </div>
 
         </fieldset>
+        <div class="box">
+          NOTE: Before you proceed, please ensure that the required queue is created on the message router vpn.
```
Comment (TamimiGitHub) [inline src/features/QueueConsumer/QueueConsumer.html:147]: can we replace `message router vpn` with `Solace PubSub+ Event Broker` throughout the samples please.

**Review by TamimiGitHub — APPROVED** [review]
LGT with minor logging modifications. Thanks!

File: src/features/QueueConsumer/QueueConsumer.html (hunk 2 above)
Comment (gvensan) [reply src/features/QueueConsumer/QueueConsumer.html]: Done!

File: src/basic-samples/GuaranteedPublisher/GuaranteedPublisher.js (hunk 1 above)
Comment (gvensan) [reply src/basic-samples/GuaranteedPublisher/GuaranteedPublisher.js:47]: Done

**Pull request merged**